doc/source/examples/protoselect_adult_cifar10.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<p>"and whom have a partner" -&gt; "and who have a partner"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>